cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -650,6 +656,56 @@ private void setAndTestRuntimeArgs(String namespace, String appId, String runnab
 
   @Category(XSlowTests.class)
   @Test
+  public void testMultipleWorkflowInstances() throws Exception {
```

Comment:
is it possible just to test the method you changed in ```ScheduleTaskRunner```?  I don't like this test because it is always very slow, and because it is completely timing based.  You never know on a slow machine (like our build machines) how it will behave.